flink-connectors/flink-connector-kinesis/src/main/resources/META-INF/NOTICE

```diff
@@ -6,7 +6,7 @@ The Apache Software Foundation (http://www.apache.org/).
 
 This project bundles the following dependencies under the Apache Software License 2.0. (http://www.apache.org/licenses/LICENSE-2.0.txt) 
 
-- com.amazonaws:amazon-kinesis-client:1.14.1
+- com.amazonaws:amazon-kinesis-client:1.14.7
```

Comment:
Did you verify there are no new transient dependencies that need to be declared or excluded?

Reply:
Good point @dannycranmer 

When running `mvn compile dependency:tree` on `master`, this is returned for the Kinesis Connector:

```
[INFO] +- com.amazonaws:amazon-kinesis-client:jar:1.14.1:compile
[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
[INFO] |  \- commons-logging:commons-logging:jar:1.1.3:compile
```

When validating it on this branch, it returns:

```
[INFO] +- com.amazonaws:amazon-kinesis-client:jar:1.14.7:compile
[INFO] |  +- org.apache.commons:commons-lang3:jar:3.3.2:compile
[INFO] |  +- commons-logging:commons-logging:jar:1.1.3:compile
[INFO] |  \- com.fasterxml.jackson.core:jackson-core:jar:2.13.0:compile
```

I'll add `com.fasterxml.jackson.core:jackson-core` to the NOTICE file.